packages/paste-website/src/components/site-wrapper/sidebar/sidebar-disclosure/SidebarDisclosureContext.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import type {DisclosurePrimitveStateReturn} from '@twilio-paste/disclosure-primitive';
+
+interface SidebarDisclosureContextProps {
+  disclosure: DisclosurePrimitveStateReturn;
+}
+
+export const SidebarDisclosureContext = React.createContext<SidebarDisclosureContextProps>({} as any);
```

Comment:
Could we replace `any` with the interface of the context? Using `any` [causes the compiler to ignore type checks.](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-explicit-any.md)

Reply:
Updated!